spaceline-all-the-icons-segments.el

```diff
@@ -1014,7 +1014,7 @@ available updates then restores the current buffer."
                      'face `(:height ,(spaceline-all-the-icons--height 1.1) :family ,(all-the-icons-octicon-family) :inherit)
                      'display '(raise 0.1))))
 
-      (propertize " " 'display '(space . (:width 50))))
```

Comment:
This was a _"hack"_ added to fix issues with miscalculated widths and gaps at the end of the mode line... I know that it causes problems elsewhere so I'm happy for this to be removed for now but it's worth noting that I may need to revisit which solution is adding/removing the most value :+1: